src/cmd.js

```diff
@@ -132,12 +126,12 @@ function _cmd(options, command, commandArgs, userOptions) {
     stdout = result.stdout || '';
     stderr = result.stderr ||
              `'${command}' encountered an error during execution`;
-    code = typeof result.code === 'number' && result.code > 0 ? result.code : 1;
+    code = result.exitCode !== undefined ? result.exitCode : 1;
```

Comment:
This needs to have `> 0` in there. It's possible to reach this branch with `exitCode === 0`, however we need to always exit this branch with `code > 0` in order to signal that the command should fail. I believe this will fail CI without the `> 0` bit.

See the code snippet that I had in the previous PR.

Reply:
```suggestion
    code = result.exitCode !== undefined && result.exitCode > 0 ? result.exitCode : 1;
```